Andres Lopez / Our Vision Grid

AndresAtUCR opened the pull request:
I got most of the format down, but I can't seem to get the horizontal spacing for the first grid (the image grid) to get closer. Any help and tips would help.

![Screenshot 2025-04-20 185321](https://github.com/user-attachments/assets/90153785-19dc-4f95-9428-6e98425baf35)

AndresAtUCR committed:
Andres Lopez / Our Vision Grid

stanleylew5 commented:
Add a tag

swedenagu commented on src/app/about/page.tsx:
```diff
@@ -1,8 +1,10 @@
 const Page = () => {
   return (
     <div>
       <Header />
```
add `text-center mt-8` to Header component inner div

AndresAtUCR replied:
From what I know I'm not supposed to change formatting on header?

swedenagu replied:
Nah you can go in and change it in the Header component

swedenagu reviewed (commented):
comments

swedenagu commented on src/components/about/OurVision.tsx:
```diff
@@ -0,0 +1,34 @@
+          university student or as a transfer student.
+        </p>
+      </div>
+    </div>
```
```suggestion
    <div className="grid grid-cols-2 ml-16">
      <div className="grid grid-cols-5">
        <div className="col-span-2 grid grid-rows-3 gap-4">
          <div className="relative aspect-[1/1] mt-4 w-auto overflow-hidden rounded-xl">
            <Image src={placeholder} alt="Placeholder Image 1" fill />
          </div>
          <div className="row-span-2 relative w-auto h-[55.5%] overflow-hidden rounded-xl">
            <Image src={placeholder} alt="Placeholder Image 2" fill />
          </div>
        </div>
        <div className="col-span-3 grid grid-rows-2 m-4 gap-4">
          <div className="relative aspect-[1/1] w-auto overflow-hidden rounded-xl">
            <Image src={placeholder} alt="Placeholder Image 3" fill />
          </div>
          <div className="relative aspect-[16/9] mt-[-48] w-auto overflow-hidden rounded-xl">
            <Image src={placeholder} alt="Placeholder Image 4" fill />
          </div>
        </div>
      </div>
      <div className="text-acm-gray-500 flex flex-col items-center text-center">
        <h1 className="w-1/2 text-3xl font-bold">OUR VISION</h1>
        <p className="mt-12 w-2/3 text-2xl font-medium">
          Our Vision is simple: We want to improve our members professionally,
          technically, and personally. We want to help our members grow
          throughout their college career, whether they enter as a first-year
          university student or as a transfer student.
        </p>
      </div>
    </div>
```

AndresAtUCR replied:
Oh wait, this is actually very smart to make more rows to give it that shape ngl. I understand it now.

AndresAtUCR committed:
Updated Grid Layout

Accepted incoming changes for the grid layout by the lead, format now fits that of the Figma.

Co-authored-by: Sweden <84001329+swedenagu@users.noreply.github.com>
Signed-off-by: Andres Lopez <alope759@ucr.edu>

AndresAtUCR committed:
Updated formatting with the help of lead suggested changes.

AndresAtUCR committed:
Merge branch 'AndresAtUCR/OurVisionGrid' of https://github.com/acm-ucr/acm-atlas into AndresAtUCR/OurVisionGrid

AndresAtUCR committed:
Updated formatting via suggested changes by lead.

AndresAtUCR commented:
![image](https://github.com/user-attachments/assets/4d01490a-3a95-4f8e-9a82-378a5d01aef0)

AndresAtUCR committed:
Merge branch 'dev' of https://github.com/acm-ucr/acm-atlas into AndresAtUCR/OurVisionGrid

AndresAtUCR committed:
Corrected formatting and added alt text.

AndresAtUCR committed:
Merge 87f80c6fbb86d5338874c586a1506876e3ecf895 into f9ae7244cbaced02750bddbea5ea5a29c4f095cc

Commit:
convert images to webp

AndresAtUCR commented:
![image](https://github.com/user-attachments/assets/7d01907a-dc76-458b-b5ab-e68c76a79d0a)

stanleylew5 reviewed (commented):
Rename photos to OurVision1 OurVision2 OurVision3 OurVision4, you don't need a folder for OurVision since there's only four photos

stanleylew5 commented on src/components/about/OurVision.tsx:
```diff
@@ -0,0 +1,57 @@
+        </div>
+      </div>
+      <div className="text-acm-gray-500 flex flex-col items-center text-center">
+        <h1 className="w-auto text-4xl font-bold">OUR VISION</h1>
```
p tags

stanleylew5 commented on src/components/about/OurVision.tsx:
```diff
@@ -0,0 +1,57 @@
+
+const OurVision = () => {
+  return (
+    <div className="ml-16 grid grid-cols-2">
```
what you can do is instead of ml-16, just restrict the width for like w-10/12 and then do mx-auto

stanleylew5 commented on src/components/about/OurVision.tsx:
if the photos look warped, crop them yourself and put it back in

stanleylew5 commented on src/components/about/OurVision.tsx:
```diff
@@ -0,0 +1,57 @@
+              src={"AboutUs/OurVision/1_1.jpg"}
+              alt="ACM Members attend an in-progress general ACM meeting."
+              fill
+              quality={85}
```
remove this

stanleylew5 commented on src/components/about/OurVision.tsx:
```diff
@@ -0,0 +1,57 @@
+        <div className="col-span-2 grid grid-rows-3 gap-6">
+          <div className="relative mt-4 aspect-[1/1] w-auto overflow-hidden rounded-xl">
+            <Image
+              src={"AboutUs/OurVision/1_1.jpg"}
```
import images at the top so like 
import OurVision1 from "@public/about/ourvision1" or something like that 

then you can have src={OurVision1}

stanleylew5 replied:
do this for all the images below

stanleylew5 commented on src/components/about/OurVision.tsx:
```diff
@@ -0,0 +1,57 @@
+              src={"AboutUs/OurVision/1_2.jpg"}
+              alt="ACM members and UCR students alike attend an ACM workshop on the basics of Python programming at the Pentland Hills Foxhole."
+              fill
+              quality={85}
```
```suggestion
```

stanleylew5 commented on src/components/about/OurVision.tsx:
```diff
@@ -0,0 +1,57 @@
+              src={"AboutUs/OurVision/1_3.jpg"}
+              alt="A screen presenting ACM VP Jerry Li's LinkedIn is provided to members as the exemplar."
+              fill
+              quality={85}
```
```suggestion
```

AndresAtUCR committed:
Merge pull request #929 from acm-ucr/acm-ucr/images_db8824341f07354f

Update Images to Webp

AndresAtUCR committed:
Updated photos & code styling.

AndresAtUCR commented:
![Screenshot 2025-04-21 205905](https://github.com/user-attachments/assets/18e5444c-5d52-42f0-a0bf-4aa9ab4e5f49)

stanleylew5 commented on src/components/about/OurVision.tsx:
```diff
@@ -0,0 +1,61 @@
+
+const OurVision = () => {
+  return (
+    <div className="mx-auto grid w-11/12 grid-cols-2">
```
constrict the width even more there's too much space in between the text and images

stanleylew5 replied:
do it until it matches the figma

AndresAtUCR committed:
Updated spacing between photos & text.

AndresAtUCR committed:
FIxed up image spacing a bit.

AndresAtUCR committed:
Merge 0dc4cd7923e43f254b3525193dc90896886e917c into 5368a0ce5ee4891b706faed5964eedb000c7ceb5

AndresAtUCR commented:
![image](https://github.com/user-attachments/assets/9ab8e3cb-853b-4121-a4b3-bccd95dd6ee3)

Commit:
convert images to webp

stanleylew5 committed:
Merge pull request #936 from acm-ucr/acm-ucr/images_cd2894da5c99739d

Update Images to Webp

stanleylew5 reviewed (changes requested):
pull and solve your merge conflicts. rename your images to like forge1 instead of forge_1. then i'll look over the rest

AndresAtUCR committed:
Merge branch 'dev' of https://github.com/acm-ucr/acm-atlas into AndresAtUCR/OurVisionGrid

AndresAtUCR committed:
Renamed photo files and tweaked OurVision Section for symmetry.

AndresAtUCR commented:
![Screenshot 2025-04-22 171530](https://github.com/user-attachments/assets/8e123fb6-5c0c-41da-ae7c-7a2880f41631)

stanleylew5 commented on public/AboutUs/OurVision2.webp:
compress this photo until its around sub 500 kb

stanleylew5 commented on public/forge1.webp:
delete file

stanleylew5 commented on public/AboutUs/OurVision1.webp:
rename em to ourvision1 and so forth

AndresAtUCR replied:
I don't even know why these didn't update, I renamed them and they were updated on my machine, I'll try again.
![image](https://github.com/user-attachments/assets/344a3209-3f0e-49a1-ab46-a5ae7fafd5df)

AndresAtUCR replied:
Gonna try a different naming convention cause vscode is being stupid

stanleylew5 commented on public/DAS2.webp:
delete file

stanleylew5 commented on public/create1.webp:
delete file

stanleylew5 commented on public/spark1.webp:
delete file

stanleylew5 commented on src/components/about/OurVision.tsx:
```diff
@@ -0,0 +1,61 @@
+import Image from "next/image";
+const OurVision1 = "AboutUs/ourvision1.webp";
```
you're importing these images wrong it should say

``import ourvision1 from "@/public/about/ourvision1.webp"``

stanleylew5 commented:
also lowercase the name of your ourvision file

AndresAtUCR committed:
Naming convention change; please disregard.

AndresAtUCR committed:
Compressed image 2 & reworked naming conventions.

AndresAtUCR commented:
Styling remains unchanged, refer to last screenshot.

stanleylew5 reviewed (changes requested):
crop the images 😭 like until they match the figma. you can use [pixlr](https://pixlr.com/express/) to crop it so it matches the figma that way you don't need overflow hidden, fill, aspect, and all that. you also didn't rename the ourvision component file to lowercase

AndresAtUCR committed:
Updated formatting.

AndresAtUCR commented:
![image](https://github.com/user-attachments/assets/2d41cdd2-caa5-4f16-9e81-725004523c21)

AndresAtUCR committed:
Fix

AndresAtUCR committed:
Empty Commit

AndresAtUCR committed:
Ran npm build, PLEASE WORK.

AndresAtUCR committed:
Reimported OurVision component.

AndresAtUCR committed:
Changed file name temporarily.

AndresAtUCR committed:
Updated file name for build.

stanleylew5 committed:
remove unneeded div tags
